cmake_modules/KratosBoost.cmake

```diff
@@ -1,23 +1,29 @@
 # Finding and including the BOOST library (version should not matter anymore)
 
-# Check if Boost_INCLUDE_DIRS and Boost_LIBRARY_DIRS are already defined as environment variables
-if(DEFINED ENV{Boost_INCLUDE_DIRS})
-  set(Boost_INCLUDE_DIRS $ENV{Boost_INCLUDE_DIRS})
-endif(DEFINED ENV{Boost_INCLUDE_DIRS})
-if(DEFINED ENV{Boost_LIBRARY_DIRS})
-  set(Boost_LIBRARY_DIRS $ENV{Boost_LIBRARY_DIRS})
-endif(DEFINED ENV{Boost_LIBRARY_DIRS})
```

Comment:
While don't specially like them, we probably need to maintain those options as well in case someone is still using them. (The ones with non capital `boost`i mean)

Reply:
It is preserved later, take a look at the whole file, the boost dependency management is super inconsistent.

Reply:
a ok, didn't saw it sorry